scripts/main/view.js

```diff
@@ -580,7 +580,7 @@ view.album = {
 	},
 
 	sidebar: function () {
-		if ((visible.album() || !album.json.init) && !visible.photo()) {
+		if ((visible.album() || (album.json !== null && album.json.init !== null)) && !visible.photo()) {
```

Comment:
can't this be simplified in:
```suggestion
		if ((visible.album() || (album.json && album.json.init)) && !visible.photo()) {
```

Reply:
@evoludolab can you check that one?
Otherwise, this is a really good job. Ready to merge IMHO.